src/lib.rs

```diff
@@ -1096,7 +1102,6 @@ pub fn prefresh(pad: WINDOW, pmin_row: i32, pmin_col: i32, smin_row: i32, smin_c
 { unsafe { ll::prefresh(pad, pmin_row, pmin_col, smin_row, smin_col, smax_row, smax_col) } }
 
 
-#[deprecated(since = "5.98.0", note = "printw format support is disabled. Use addstr instead")]
```

Comment:
Why was this removed?

Reply:
That's a good question.
It seems to me it was deprecated while it was vulnerable because any `%` in the user-provided text(ie. the function arg) would be treated as formatting by the underlying code but since it was missing the extra args for the reference params, it would segfault.

But since that got fixed [here](https://github.com/jeaye/ncurses-rs/commit/8fd11147ec9cc85d07a21f71afd07874e06f4a5f#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L1078-R1091) before my PR by someone else, it seemed to me there's no good reason to deprecate it anymore, so I had it undeprecated in this PR commit [here](https://github.com/jeaye/ncurses-rs/pull/218/commits/226b8467aa2af976e02bb55a314e709472efd51b)

Reply:
I figured as much. Makes sense to me. Thanks!